src/translations/en.json

```diff
@@ -62,6 +62,20 @@
         "12": "Your other cards may only be targeted by scuttles",
         "13": "Reduces the points you need to win. (1K: 14pts, 2K: 10pts, 3K: 5pts, 4K: 0pts)"
       }
+    },
+    "dialogs":{
```

Comment:
The `game` block already has a `dialogs` block, starting at line 39

Reply:
Ah! Yes, my bad. This must have been there after I updated my branch to be in sync with the main branch. Thanks for pointing this out as I didn't check the files after the sync.